emukit/core/optimization/gradient_acquisition_optimizer.py

```diff
@@ -0,0 +1,81 @@
+from typing import Tuple
+
+import GPyOpt
+import numpy as np
+
+from .. import ParameterSpace
+from ..acquisition import Acquisition
+from .acquisition_optimizer import AcquisitionOptimizerBase
+
+
+import logging
+_log = logging.getLogger(__name__)
+
+
+class GradientAcquisitionOptimizer(AcquisitionOptimizerBase):
+    """ Optimizes the acquisition function """
```

Comment:
I just realized, that you could also pass the `optimizer` argument and use DIRECT or CMA with GPyOpt instead of L-BFGS. Both  DIRECT/CMA are gradient-free. Therefore this optimizer might be poorly named. What these three have in common, is that they expect a continuous search space. I see three options, which one do you prefer?
1. Rename to `ContinuousAcquisitionOptimizer`
2. Rename to `GPyOptAcquisitionOptimizer`
3. Force use of L-BFGS.

Reply:
Direct and CMA are not stable in GPyOpt, let's not even give the users a way to use them. So option 3, thanks for checking